### PR TITLE
Docker queries

### PR DESCRIPTION
**Pull request by ne-grebe**

### CODE REVIEW

File: models/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Object.keys(db).forEach(modelName => {
+  if (db[modelName].associate) {
+    db[modelName].associate(db);
+  }
+});
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
+
```
Comment (lawhorkl) [inline models/index.js]: Somehow compliled JS made it into your PR

File: src/Repositories/TodoRepository.ts [hunk 2]
```diff
@@ -13,79 +11,57 @@ export interface Todo {
...
-            ...requestedTask[0], ...updatePayload
-        }
-
-        const position = this.taskArray.indexOf(requestedTask[0])
-        this.taskArray[position] = updatedTask
-
-        return updatedTask;
+    async remove(todoID: string) {
+        const postTodoRemovalList = Connection.deleteTodo(todoID)
+        return postTodoRemovalList
```
Comment (lawhorkl) [inline src/Repositories/TodoRepository.ts]: Why the added layer between Sequelize and the repo?

**Review by lawhorkl — CHANGES_REQUESTED** [review]

File: models/index.js (hunk 1 above)
Comment (ne-grebe) [reply models/index.js]: pretty sure this was autogenerated when I started sequelize; my stuff works without this file

File: src/Repositories/TodoRepository.ts (hunk 2 above)
Comment (ne-grebe) [reply src/Repositories/TodoRepository.ts]: I was getting some error with the interfaces in TodoRepository.ts a few days ago whenever I tried to establish a connection in that file... But I just tried to combine them now and it works just fine.

File: src/Repositories/TodoRepository.ts [hunk 3]
```diff
@@ -13,79 +11,152 @@ export interface Todo {
...
+            `
+            INSERT INTO 
+                Todos 
+            VALUES (
+                "${todoToAdd.id}", 
+                "${todoToAdd.createdAt}", 
+                "${todoToAdd.name}", 
+                "${todoToAdd.dueDate}", 
+                "0"
+            )`)
```
Comment (lawhorkl) [inline src/Repositories/TodoRepository.ts:51]: I left comments about this on Chase's PR as well.

Allowing user input directly into SQL queries is very dangerous and makes you vulnerable to a [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) attack.

You can get around this vulnerability by [sanitizing](https://www.quora.com/What-exactly-is-data-sanitization-with-respect-to-SQL-injection) user input in all cases.

From my reading the best practice way to do this with Hapi is through either [replacements](https://sequelize.org/v7/manual/raw-queries.html#replacements) or [bind parameters](https://sequelize.org/v7/manual/raw-queries.html#bind-parameter)

File: src/Repositories/TodoRepository.ts [hunk 4]
```diff
@@ -13,79 +11,152 @@ export interface Todo {
...
+        
+        //will find and return all todos with the same bit value as determined above
+        const [results, metadata] = await instance.dbs.todoDB.sequelize.query(
+            `
+            SELECT * 
+            FROM Todos 
+            WHERE todo_completed_status = ${placeholderBool}
+            `)
+        return results
     }
```
Comment (lawhorkl) [inline src/Repositories/TodoRepository.ts:95]: good job on recognizing that this doesn't need to be two functions

File: src/Repositories/TodoRepository.ts (hunk 3 above)
Comment (lawhorkl) [reply src/Repositories/TodoRepository.ts:51]: Fully preventing SQL injection attacks are also out of scope for this training this is primarily meant as an FYI in the future

**Review by lawhorkl — APPROVED** [review]
Good job.

**Pull request merged**